apis/core/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -166,56 +166,129 @@ func (r *OpenStackControlPlane) ValidateServices(basePath *field.Path) field.Err
 			err := field.Invalid(basePath.Child("keystone").Child("enabled"), r.Spec.Keystone.Enabled, depErrorMsg)
 			allErrs = append(allErrs, err)
 		}
+
+		if isUpdate {
+			if err := r.Spec.Keystone.Template.ValidateUpdate(basePath.Child("keystone").Child("template")); err != nil {
```

Comment:
This signature is problematic. The original ValidateUpdate signature from the webhook has the signature of  `ValidateUpdate(old runtime.Object)` so webhooks can compare old and new values during update validation. So we need to pass such old values from openstack-operator too to keep the same possibility